sdk/easm/Azure.Analytics.Defender.Easm/README.md

```diff
@@ -0,0 +1,168 @@
+
+## Getting started
+### Install the package
+
+First, provide instruction for obtaining and installing the package or library. This section might include only a single line of code, like `dotnet add package package-name`, but should enable a developer to successfully install the package from NuGet, npm, or even cloning a GitHub repository.
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/ ):
+
+```dotnetcli
+dotnet add package Azure.Analytics.Defender.Easm --prerelease
+```
+
+### Prerequisites
+> You must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/) and [EASM workspace](https://learn.microsoft.com/azure/external-attack-surface-management/deploying-the-defender-easm-azure-resource). In order to take advantage of the C# 8.0 syntax, it is recommended that you compile using the [.NET Core SDK](https://dotnet.microsoft.com/download) 3.0 or higher with a [language version](https://docs.microsoft.com/dotnet/csharp/language-reference/configure-language-version#override-a-default) of `latest`.  It is also possible to compile with the .NET Core SDK using a language version of `preview`.
+
+### Authenticate the client
+
+The examples shown below use a [`DefaultAzureCredential`][DefaultAzureCredential], which is appropriate for most scenarios including local development and production environments utilizing managed identity authentication.
+
+You can find more information on different ways of authenticating and their corresponding credential types in the [Azure Identity][azure_identity] documentation.
+
+To use the `DefaultAzureCredential` provider shown below,
+or other credential providers provided with the Azure SDK, you must first install the Azure.Identity package:
+
+```dotnetcli
+dotnet add package Azure.Identity
+```
+
+To create an EasmClient, you need your subscription ID, region, and some sort of credential. Below is an example using the DefaultAzureCredential.
+
+```C# Snippet:Sample1_AssetResources_Create_Client
+            string endpoint = "https://<region>.easm.defender.microsoft.com/subscriptions/<Your_Subscription_Id>/resourceGroups/<Your_Resource_Group_Name>/workspaces/<Your_Workspace_Name>";
+EasmClient client = new EasmClient(new System.Uri(endpoint),
+                new DefaultAzureCredential());
+```
+
+## Key concepts
+
+### [Assets][assets_documentation]
+Defender EASM includes the discovery of the following kinds of assets:
```

Comment:
```suggestion

Defender EASM includes the discovery of the following kinds of assets:
```

Adding one comment here, but all header sections should be followed by a blank line, according to the Microsoft style guide.